### PR TITLE
adaptation to new mwe_query version

### PR DESCRIPTION
**Pull request by bbonf**
note: should also revisit #46 when fixing this

### CODE REVIEW

**Comment by bbonf** [comment]
Exclusion queries don't work well because of missing index expansion

File: backend/mwe/views.py [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
+log = logging.getLogger(__name__)
+
 
 class CanonicalFormList(ListAPIView):
     queryset = CanonicalForm.objects.all()
     serializer_class = CanonicalFormSerializer
 
 
-def generate_query(sentence, rank):
+def generate_queries(sentence):
```
Comment (tijmenbaarda) [inline backend/mwe/views.py:22]: Perhaps add type hints

**Comment by tijmenbaarda** [comment]
sastadev should be added to the requirements

**Comment by bbonf** [comment]
> sastadev should be added to the requirements
It's a requirement for mwe-query, but it was not included here because it's not in mwe-query@develop yet

**Pull request merged**